openshift/packit-worker.yml.j2

```
@@ -65,6 +65,8 @@ spec:
               value: {{ distgit_url }}
             - name: DISTGIT_NAMESPACE
               value: {{ distgit_namespace }}
+            - name: PACKIT_USER_AGENT
+              value: "packit-ci/{{ project }} ({{ deployment }}+{{ project }}@packit.dev)"
```

```suggestion
              value: "packit.dev/{{ service }}-{{ deployment }} ({{ deployment }}+{{ service }}@packit.dev)"
```

Which should result in something like: 

```
packit.dev/packit-prod (prod+packit@packit.dev)
packit.dev/fedora-source-git-prod (prod+fedora-source-git@packit.dev)
```

and avoid the "double-prod" in the email part.